scipy/integrate/meson.build

```diff
@@ -160,6 +150,7 @@ py3.extension_module('_lsoda',
   link_with: [lsoda_lib, mach_lib],
   c_args: [numpy_nodepr_api, '-Wno-unused-variable'],
   dependencies: [py3_dep, lapack],
+  c_args: [numpy_nodepr_api, Wno_unused_variable],
```

Comment:
this should replace the `c_args` above, the string style was superseded in backports IIRC